perspective_mapping.py

```diff
@@ -13,8 +13,8 @@ def nothing(x):
 cv2.createTrackbar('alpha','image',0,360,nothing)
 cv2.createTrackbar('beta','image',0,360,nothing)
 cv2.createTrackbar('gamma','image',0,360,nothing)
-cv2.createTrackbar('f','image',0,2000,nothing)
-cv2.createTrackbar('z_translation','image',0,2000,nothing)
+cv2.createTrackbar('f','image',1,20,nothing)
```

Comment:
Hey I like that defaults are now not 0 but I think we should increase the focal length maximum back to 2000. This is because as a rule of thumb, the focal length of a random camera is about the same number as the height of an image (ie like 480 px or like 1200 depending on the resolution and FOV).
Images with a higher FOV will generally have a smaller focal length compared to a zoomed in shot with a long focal length.

Reply:
Oh and spacing of the pixels on the camera sensor is also a factor in focal length